src/components/TodoItem.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/prop-types */
+/** @format */
```

Comment:
- [ ] Just as I stated above, kindly endeavor to not disable linters when you are writing your code because they are meant to check for best practices which helps in making you a better developer in following the industry’s standard of writing clean code 👍.